src/main/java/seedu/address/logic/commands/eventvolunteercommands/EventAddVolunteerCommand.java

```diff
@@ -63,11 +68,40 @@ public CommandResult execute(Model model) throws CommandException {
         if (eventToAssign.hasVolunteer(volunteerToAssign)) {
             throw new CommandException(MESSAGE_DUPLICATE_VOLUNTEER);
         }
+
+        Set<EventName> assignedEvents = volunteerToAssign.getAssignedEvents();
+        for (EventName eventName : assignedEvents) {
+            Predicate<Event> eventPredicate = e -> e.getEventName().equals(eventName);
```

Comment:
doesn't seem like this `eventPredicate` is used?

Reply:
thats right